config.md

```diff
@@ -843,6 +846,7 @@ Here is a full example `config.json` for reference.
 [selinux]:http://selinuxproject.org/page/Main_Page
 [no-new-privs]: https://www.kernel.org/doc/Documentation/prctl/no_new_privs.txt
 [proc_2]: https://www.kernel.org/doc/Documentation/filesystems/proc.txt
+[umask.2]: http://pubs.opengroup.org/onlinepubs/009695399/functions/umask.html
```

Comment:
nit: POSIX functions should go be in section 3 (or 3p), so `umask.2` should be `umask.3`.  `umask.2` would be [the Linux kernel docs for `umask`][1].

nit: you're linking to the 2004 edition of POSIX.  I'd rather stay consistent with our other links and use [the 2016 edition][2] (#858).

[1]: http://man7.org/linux/man-pages/man2/umask.2.html
[2]: http://pubs.opengroup.org/onlinepubs/9699919799/functions/umask.html